epsagon/event.py

```diff
@@ -105,3 +107,5 @@ def set_exception(self, exception, traceback_data):
         self.exception['message'] = str(exception)
         self.exception['traceback'] = traceback_data
         self.exception['time'] = time.time()
+        if not handled:
```

Comment:
Wouldn't it be a bit more clear to call it "manually_set" instead of handled/unhandled?

Reply:
I think that 'handled' it's good. It can be 'handled' and not manually set - for example, if we will extract errors from calling `logging.error`